yarn.lock

```diff
@@ -1743,15 +1743,20 @@
     shortid "~2.2.15"
     ts-key-enum "^2.0.0"
 
-"@itwin/core-bentley@3.2.0", "@itwin/core-bentley@^3.2.0":
+"@itwin/core-bentley@3.2.0":
   version "3.2.0"
```

Comment:
Why is this stuck to 3.2.0 ?

Reply:
Hmm, related to core-geometry which is at 3.0.0... Weird.

Reply:
for some reason thats ver getting pulled inc core-geom@3.0.0 which is getting pulled in from reality-data-client

Reply:
because realitydata client is ^0.7.2, so it does not pull the latest version, which is 0.9.0... Should we update to require 0.9.0 ?

Reply:
yea im going to go pull in the latest viewer and all the latest deps possible, give me a few mins

Reply:
had to manually dedupe this with a reoslution, revert the resolution but it should be ok now.

might be an issue with yarn